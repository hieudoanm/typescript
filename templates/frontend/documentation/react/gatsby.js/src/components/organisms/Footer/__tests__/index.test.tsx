import { render } from '@testing-library/react';
import { Footer } from '../index';
import React from 'react';

describe('Footer', () => {
  it('renders correctly', () => {
    const button = render(<Footer title="" socialLinks={[]} />);
    expect(button.container).toMatchSnapshot();
  });
});
