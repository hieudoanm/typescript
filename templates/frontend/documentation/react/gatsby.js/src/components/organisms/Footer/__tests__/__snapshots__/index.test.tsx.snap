// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <footer
    class="border-t py-8 md:border-none"
  >
    <div
      class="container mx-auto"
    >
      <div
        class="flex items-center justify-between"
      >
        <div
          class="font-lg font-medium"
        >
          Â© 
          2024
           
        </div>
        <div
          class="flex items-center gap-2 md:gap-4"
        />
      </div>
    </div>
  </footer>
</div>
`;
